spec/models/comment_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
+require 'spec_helper'
+require 'support/shared_examples'
 
-RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+describe Comment, type: :model do
+  describe 'validations' do
+    let(:comment) { build :comment }
```

Comment:
Is this necessary since you have `let(:resource) { create(described_class) }` in the shared_examples? Goes for all other tests like this, too